cypress/pageObjects/components/CommonElements.ts

```diff
@@ -0,0 +1,83 @@
+import {BasePage} from "../BasePage";
+
+const HEADER_SEARCH_BAR = "[data-cy=search-input-base]"
+const MODAL_SEARCH_INPUT = "#outlined-search"
+const ADDRESS_BOOK_ENTRIES = "[data-cy=address-book-entry] span a"
+const ADDRESS_BOOK_NAME_INPUT = "#name"
+const ADDRESS_BOOK_SAVE = "[data-cy=address-save]"
+const ADDRESS_BOOK_CANCEL = "[data-cy=address-cancel]"
+const ADDRESS_BOOK_REMOVE = "[data-cy=address-remove]"
+const ALL_ACCOUNT_SEARCH_RESULTS = "[data-cy*=account-search-result]"
+const SUBGRAPH_BUTTON = "#search-button"
+const SUBGRAPH_CONTAINER = ".graphiql-container"
+
+export class CommonElements extends BasePage {
+
+
+  static clickHeaderSearchBar() {
+    cy.get(HEADER_SEARCH_BAR).parent().click()
+    //this.click(HEADER_SEARCH_BAR)
+  }
+
+  static searchForStaticBalanceAccount() {
+    cy.fixture("commonData").then(fixture => {
+      this.type(MODAL_SEARCH_INPUT, fixture.staticBalanceAccount)
+    })
+  }
+
+  static openNetworkAddressResult(network: string) {
+    cy.get("[data-cy=" + network + "-account-search-result]").then(el => {
+      cy.wrap(el.text()).as("lastOpenedResult")
+    })
+    this.click("[data-cy=" + network + "-account-search-result]")
+  }
+
+  static validateAccountPageLink(network: string) {
+    cy.get("@lastOpenedResult").then(result => {
+      this.validatePageUrl("/" + network + "/accounts/" + result + "?tab=streams")
```

Comment:
You don't prefer string interpolation, as in: ``` `/${network}/accounts/${result}?tab=streams` ```?

Reply:
Latvian keyboard settings always mess up the ` character , e.g. if I end this string I would get **streamš** instead of **streams`**  so I use +